### DISCUSSION TITLE
VSCode extension with remark-directive gives error

### DISCUSSION THREAD

**Discussion by subproject22** [mdx-js/mdx#2543] [Q&A]
Using MDX extension, remark-directive and simple React component. Everything working, but it shows error in VSCode when I pass multiple directives. `Could not parse expression with acornMDXmicromark-extension-mdx-expression:acorn`

<img width="391" alt="Screenshot 2024-09-29 at 15 40 31" src="https://github.com/user-attachments/assets/64764989-506f-4c0e-a9b8-9570aa79bc0c">

Any way to silent or fix it? Tried to setup tsconfig, no effect

```
"mdx": {
    "plugins": ["remark-directive"]
  }
 ```

**Comment by wooorm** [comment]
That looks like a problem with the extension / language server. That can be reporter at https://github.com/mdx-js/mdx-analyzer
